tests/puppeteer/campaigns/functional/BO/03_catalog/05_brandsAndSuppliers/suppliers/04_sortSuppliers.js

```diff
@@ -0,0 +1,176 @@
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+const loginCommon = require('@commonTests/loginBO');
+// Importing data
+const SupplierFaker = require('@data/faker/supplier');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const BrandsPage = require('@pages/BO/catalog/brands');
+const SuppliersPage = require('@pages/BO/catalog/suppliers');
+const AddSupplierPage = require('@pages/BO/catalog/suppliers/add');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_catalog_brandsAndSuppliers_suppliers_sortSuppliers';
+
+let browser;
+let page;
+const firstSupplierData = new SupplierFaker();
+const secondSupplierData = new SupplierFaker();
+let numberOfSuppliers = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    brandsPage: new BrandsPage(page),
+    suppliersPage: new SuppliersPage(page),
+    addSupplierPage: new AddSupplierPage(page),
+  };
+};
+
+// Sort suppliers table
+describe('Create, update and delete supplier', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+    await Promise.all([
+      files.deleteFile(firstSupplierData.logo),
+      files.deleteFile(secondSupplierData.logo),
+    ]);
+  });
+
+  // Login into BO
+  loginCommon.loginBO();
+
+  // Go to brands page
+  it('should go to brands page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToBrandsPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.catalogParentLink,
+      this.pageObjects.boBasePage.brandsAndSuppliersLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.brandsPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.brandsPage.pageTitle);
+  });
+
+  // Go to suppliers page
+  it('should go to suppliers page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToSuppliersPage', baseContext);
+    await this.pageObjects.brandsPage.goToSubTabSuppliers();
+    const pageTitle = await this.pageObjects.suppliersPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.suppliersPage.pageTitle);
+  });
+  // 1: Create two suppliers
+  describe('Create 2 suppliers', async () => {
+    const tests = [
+      {args: {supplierToCreate: firstSupplierData}},
+      {args: {supplierToCreate: secondSupplierData}},
```

Comment:
You're using only 2 suppliers ? Is it enough to verify ?

Reply:
I can add more suppliers, but I think if the sort of 2 different suppliers is passed correctly also the sort of 3,4,5... will be sorted correctly.

Reply:
Could you add a third one please ? That way we're sure that a sort is correct. It's impossible to verify with only 2.

Reply:
Done :)